<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>650</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>610</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Restart</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxMode">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>280</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Tryb</string>
    </property>
    <widget class="QRadioButton" name="radioButtonManual">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>116</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Manualny</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonAuto">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Automatyczny</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxSpeed">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>170</y>
      <width>120</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Prędkość</string>
    </property>
    <widget class="QDial" name="dialSpeed">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>71</width>
       <height>64</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxView">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>90</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Widoczność</string>
    </property>
    <widget class="QRadioButton" name="radioButtonMaze">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>116</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Labirynt</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonMemory">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>116</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Pamięć</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxTime">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Kroki</string>
    </property>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>5</number>
     </property>
     <property name="smallDecimalPoint">
      <bool>true</bool>
     </property>
     <property name="value" stdset="0">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>671</width>
      <height>671</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Up">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>470</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Down">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>530</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Left">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>500</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Right">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>500</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Right</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>370</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Sciezka pliku</string>
    </property>
    <widget class="QLineEdit" name="FilePathArea">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>121</width>
       <height>29</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>824</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>radioButtonMaze</tabstop>
  <tabstop>radioButtonMemory</tabstop>
  <tabstop>dialSpeed</tabstop>
  <tabstop>radioButtonManual</tabstop>
  <tabstop>radioButtonAuto</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>startButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>radioButtonAuto</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>757</x>
     <y>644</y>
    </hint>
    <hint type="destinationlabel">
     <x>743</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>radioButtonMaze</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>773</x>
     <y>647</y>
    </hint>
    <hint type="destinationlabel">
     <x>748</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
